File: Java/1136 - Prime Generator.java
import java.util.ArrayList;
import java.util.Scanner;

public class main {

  public static void main(String arg[]) {
    Scanner l = new Scanner(System.in);
    ArrayList<String> total = new ArrayList<String>();
    int i, j;
    int a = l.nextInt();
    for (int h = 0; h < a; h++) {
      boolean esPrimo;
      int rInicial = l.nextInt();
      int rFinal = l.nextInt();
      if (rInicial <= 1) {
        rInicial = rInicial + 1;
      }
      for (i = rInicial; i <= rFinal; i++) {
        esPrimo = true;
        for (j = 2; j < i; j++) {
          if (i % j == 0) {
            esPrimo = false;
          }
        }
        if (esPrimo) {
          total.add("" + i);
        }
      }
      total.add("");
    }
    for (i = 0; i < total.size(); i++) {
      System.out.println(total.get(i));
    }
  }
}

File: c++/1136 - Prime Generator.cpp
#include <iostream>

using namespace std;

int main()
{
      ios_base::sync_with_stdio(0); cin.tie(0);
    int i,j,rInicial,rFinal,a,k;
    bool esPrimo=true;
    cin>>a;
    for(k=0;k<a;k++){
    cin>>rInicial>>rFinal;
    if(rInicial<=1){
	  rInicial=rInicial+1;
  }
  for(i = rInicial;i <= rFinal;i++){

      esPrimo=true;
       for(j = 2;j < i;j++){

       if(i % j == 0){
          esPrimo = false;
       }
       }
       if(esPrimo){
        cout<<i<<"\n";
       }

  }
  cout<<"\n";
    }
    return 0;
}
